amida-tech/amida-messaging-microservice
# [DEVOPS-390] Version bump husky. lint/test on pre-push (previously pr…

If this PR fixes a bug, you _must_ add test cases representative of the bug.

#### What's this PR do?

#### Related JIRA tickets:

#### How should this be manually tested?
To test:
- Create a dummy branch
- Make a test commit. Linting shouldn't run.
- Attempt to push to a remote dummy branch. Linting should run and fail because there are linting errors

Linting errors being addressed in DEVOPS-384

#### Any background context you want to provide?

#### Screenshots (if appropriate):
